nautobot_golden_config/filters.py

```diff
@@ -1,4 +1,4 @@
-"""Filter for Device Configuration Backup."""
+"""Filters for UI Views."""
```

Comment:
To be clear, FilterSets are also used in the REST API (thus your changes to api/views.py above).